packages/lib/modules/pool/actions/create/steps/type/ChooseNetwork.tsx

```diff
@@ -38,30 +37,39 @@ export function ChooseNetwork({ control }: { control: Control<PoolCreationForm>
         control={control}
         name="network"
         render={({ field }) => (
-          <SimpleGrid columns={2} spacing="md" w="full">
-            {networkOptions.map(network => (
-              <Card key={network} padding="0">
-                <Checkbox
-                  flexDirection="row-reverse"
-                  gap="md"
-                  isChecked={field.value === network}
-                  justifyContent="space-between"
-                  onChange={e => {
-                    if (e.target.checked) {
-                      resetPoolCreationForm()
-                      field.onChange(network)
-                    }
-                  }}
-                  padding="sm"
-                >
-                  <HStack spacing="sm">
-                    <NetworkIcon chain={network} size={8} />
-                    <Text fontWeight="bold">{capitalize(network)}</Text>
-                  </HStack>
-                </Checkbox>
-              </Card>
-            ))}
-          </SimpleGrid>
+          <RadioCardGroup
+            key={field.value ?? 'network'}
+            name={field.name}
+            onChange={(value: GqlChain) => {
+              resetPoolCreationForm()
+              field.onChange(value)
+            }}
+            options={networkOptions}
+            radioCardProps={{
+              containerProps: {
+                _checked: {
+                  borderColor: 'background.highlight',
+                  color: 'font.opposite',
+                },
+                _focus: {
+                  boxShadow: 'outline',
+                },
```

Comment:
I agree with moving away from the checkboxes for network selection, but would suggest matching the background / border style of [original design](https://www.figma.com/design/9a1hNzM4fJxAzI5dZXeI1X/Zen-Pool-Creator?node-id=55-5058) ( and reclamm config selection )

<img width="689" height="249" alt="image" src="https://github.com/user-attachments/assets/0841e8c0-6d1a-4664-b538-49c6f6127040" />

<img width="735" height="225" alt="image" src="https://github.com/user-attachments/assets/b174ba39-29ed-4a6a-9a86-aab5cae5cf38" />



<img width="616" height="246" alt="image" src="https://github.com/user-attachments/assets/3d750332-8873-4b54-b1cf-25b7df7ad92d" />

Reply:
updated the colors